RocketCast/EpisodeModel.swift

```diff
@@ -29,4 +29,14 @@ struct EpisodeModel {
         imageURL = ""
         mp3URL = ""
     }
+    
+    init(setTitle: NSMutableString) {
```

Comment:
I'm still not convinced that NSMutableString is a solution. I think it's just a stopgap from using nils. 

Based on apple documentation of swift 3, I think they would prefer that we use `String`
https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/StringsAndCharacters.html
(see the section on `String Mutability`)

There's also this section on Obj-C and NSMutable 
http://stackoverflow.com/a/7275142/3204427

@whitef0x0 I think the reason you used NSMutableString is because it's been used before. This is something that @QuantumSpark  and I discussed and debated on. 

I'm not sure how to approach this now, but I think a refactor from NSMutableString to String? is best and use `if let` and `guard` to handle `nil`